ni_measurementlink_service/session_management/_client.py

```diff
@@ -389,3 +392,16 @@ def _timeout_to_milliseconds(timeout: Optional[float]) -> int:
         return -1
     else:
         return round(timeout * 1000)
+
+
+def _to_dict(
```

Comment:
Either rewrite this function to be as generic as it sounds, or rename it to sound as specific as it is.

Reply:
FYI, if Python had overloading like C++ and C# do, I wouldn't have made this comment.

Reply:
It's available using library's:
https://pypi.org/project/multimethod/
https://pythonhosted.org/multidispatch/

Reply:
> Either rewrite this function to be as generic as it sounds, or rename it to sound as specific as it is.

I've renamed it to `_to_group_mappings_dict`.